packages/app/src/SearchTimeRangePicker.tsx

```diff
@@ -37,7 +37,7 @@ export default function SearchTimeRangePicker({
   setInputValue,
   onSearch,
   showLive = false,
-  timeFormat = '24h',
+  timeFormat = '12h',
```

Comment:
minor thing - it looks like we end up overwriting this in the UI via https://github.com/hyperdxio/hyperdx/blob/f231d1f65f3e431cc72f2d6c152109c350d19c59/packages/app/src/timeQuery.ts#L18 (so after you hit search for the time range, we end up showing 24h time again instead of 12h)

That being said, I don't think it's a huge deal, and something we can probably clean up once https://github.com/hyperdxio/hyperdx/pull/75 lands since it might conflict anyways.